sources.json

```diff
@@ -54,5 +54,47 @@
                 "source": "https://download.vvs.de/SBahn_Liniennetz.pdf"
             }
         ]
+    },
+    {
+        "shortOperatorName": "hvv-hha",
+        "contributors": [
+            {
+                "github": "echtnurich"
+            }
+        ],
+        "sources": [
+            {
+                "name": "Wikipedia U-Bahn Hamburg",
+                "source": "https://de.wikipedia.org/wiki/U-Bahn_Hamburg"
```

Comment:
I'm not happy with Wikipedia as as source. The colors are also at the "Liniennetzplatz". Can you change this?

`https://www.s-bahn-hamburg.de/fahrplan/liniennetzplan`

Reply:
It is the only source that has all lines, including upcoming ones, unfortunately.

The concepts for a new map all have slightly differing colours for the other lines, so this was the best one.

Reply:
I'm not happy with Wikipedia as a source too. Upcoming lines are for the moment not relevant, so the source should either be the hvv-USAR-Plan, which is the same as @MrKrisKrisu mentioned before.

Reply:
I've checked a couple of official sources and they are all different.

I've changed the available ones now, and lets hope that the new plan in 3 months still uses the same set of colours, i guess.

Reply:
for reference, this is the mess that i had to comb though:
![image](https://github.com/Traewelling/line-colors/assets/24839517/98f18aab-571e-411e-b22c-def671f2f234)

There is no standard, so i picked the most complete set.

Reply:
Since HAFAS line codes may change on the Fahrplanwechsel, I'll remove them for the moment and we can have a look on them in December again.